ci

```diff
@@ -139,11 +163,27 @@ docker:run() { local imageName=$1; local tag=$2; local args=${@:3}
     docker:cli run "${imageName}:${tag}" ${args}
 }
 
-dev:environment:start() {
-    local tag="$(docker:build:tag:forLocalDev)"
-    local imageName="react-express-typescript-dev-environment"
-    docker:buildAndTag ${imageName} ${tag} ./Dockerfile .
-    docker:run ${imageName} ${tag}
+bash:runInBackground() { local logFile=$1; local command=${@:2}; export clearLogs
```

Comment:
can we break up the items into several files instead of such a long file

Reply:
i like the suggestion, matty, but not right now